tests/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/ToTsvectorTest.php

```diff
@@ -20,13 +20,21 @@ protected function getExpectedSqlStatements(): array
     {
         return [
             'SELECT to_tsvector(c0_.text) AS sclr_0 FROM ContainsText c0_',
+            'SELECT to_tsvector(LOWER(c0_.text)) AS sclr_0 FROM ContainsText c0_',
+            'SELECT to_tsvector(1 + 1) AS sclr_0 FROM ContainsText c0_',
+            'SELECT to_tsvector(1) AS sclr_0 FROM ContainsText c0_',
+            'SELECT to_tsvector(LENGTH(c0_.text)) AS sclr_0 FROM ContainsText c0_',
```

Comment:
The checks from lines 24, 25 and 26 won't produce meaningful SQL, as _to_tsvector(integer)_ is not a valid function. You can safely drop those scenarios.